volttrontesting/fixtures/cert_fixtures.py

```diff
@@ -0,0 +1,77 @@
+import contextlib
+from cryptography.hazmat.primitives import serialization
+import os
+from types import SimpleNamespace
+
+from volttron.platform.certs import CertWrapper
+
+
+@contextlib.contextmanager
+def certs_profile_1(certificate_dir, fqdn=None, num_server_certs=1, num_client_certs=3):
```

Comment:
Certs class can take a certs directory as init param. Can we create a certs class instance and then use certs method to create the server and client certs instead of duplicating code?

Reply:
I wanted a specific path to be able to be set...not just the identity set so that's why I did this so that I could "know" everything about where things were going.